lib/trust-services-manager.js

```diff
@@ -227,29 +228,114 @@ class TrustServicesManager extends PkiExpressOperator {
 		});
 	}
 
+	/** 
+	 * Starts the authentication with all configured trusted service
+	 *
+	 * @param {string} redirectUrl
```

Comment:
Here, please add the commends of each parameters. You can copy the description of each parameters following the command help:
```
Command start-service-auth: Starts authorization sessions for all configured trust service providers

Usage: pkie start-service-auth <redirectUrl>

Options:
 -trst | --session-type <session_type>    The type of the session (see below)
 -slt | --session-lifetime <secs>         The time (in seconds) of the signature session
 -cs | --custom-state <state>             The state that is preserve during the session (see
                                          complete-service-auth command to see how it's recovered)

Available Trust Service session types (-trst <session_type>:
  SignatureSession    The session will allow multiple requests
  SingleSignature     The session will allow a single signature in one request.
  MultiSignature      The session will allow multiple signature in one request.
```